### ISSUE TITLE
Help with building

### ISSUE THREAD

**Issue by ypnos** [Martchus/PKGBUILDs#75]
For quite a while, I am using many of your mingw packages but also gitea and few others that saved me a lot of time building (even more so, waiting for the build). Recently we got a new compute server here that has lots of idle times and is capable of building large packages really fast (for example, `chromium-vaapi` in 33 minutes).

So we would be in a position now where we can give back and help out with building the mingw packages. Obviously a key issue is that we would need to establish your trust in my builds.

Let me know if you are interested, or if not you can also just close this issue ;). Thanks for your great work!

**Comment by Martchus** [comment]
To start it would make sense if you would just provide the packages at some location so I could download them. If they look good I would add them to the repository.

But we would need to coordinate who builds what. For packages I maintain myself it would make most sense if I also build them myself (so I can adjust things and investigate the build directory if something goes wrong).

So I think before introducing additional developers it makes sense to think about increasing the level of automation because the lack of automation is currently the greatest bottleneck. Maybe I'll continue with my build service project. If that would work like I have it in mind it could definitely benefit from having another 'build worker' and the necessity to coordinate builds manually would go away. I'm aware of [aurbs](https://github.com/asdil12/aurbs) but it doesn't fit my needs - using it would make things even slower and more inefficient (e.g. it would rebuild all mingw-w64 packages when updating mingw-w64-headers and might not be capable of handling cyclic dependencies well enough).

Also the development of the PKGBUILDs itself is much more time consuming than the build. On my TODO list right now is mingw-w64-qt5-webkit (blocks mingw-w64-icu and mingw-w64-angleproject), mingw-w64-libfilezilla, mingw-w64-filezilla, mingw-w64-postgresql, mingw-w64-mariadb-connector-c, mingw-w64-angleproject and android-qt5 using libc.

---

TL;DR So thanks for the offer, I'll think about it. But I'd like to improve the workflow first which is only one thing on my TODO list.
